src/models.py

```diff
@@ -206,7 +277,9 @@ def dice_loss(output, target, smooth=0, activation='softmax'):
     loss = 0
     dice = DiceLoss(smooth=smooth)
     output = activation_nn(output)
-    for class_nr in range(1, int(target.max()) + 1):
+    for class_nr in range(output.size(1)):
```

Comment:
@taraspiotr Thanks for fixing that. I think we didn't see it here because case where entire batch doesn't have 1 class is rare but it could easily happen in the next project.